utils.py

```diff
@@ -453,3 +454,17 @@ def get_all_language_codes_and_names():
 def unescape_encoded_uri_component(escaped_string):
     """Unescape a string that is encoded with encodeURIComponent."""
     return urllib.unquote(escaped_string).decode('utf-8')
+
+
+cache_slug = None  # pylint: disable=invalid-name
+def get_cache_slug():
```

Comment:
It's in utils_tests.py and they are testing a `get_cache_slug()` defined in utils.py.